charts/simpledataexchanger/values.yaml

```diff
@@ -0,0 +1,259 @@
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+# backend for SDE
+replicaCount: 1
+
+sdepostgresql:
+# -- PostgreSQL chart configuration
+  # Switch to enable or disable the PostgreSQL helm chart
+  enabled: true
+  fullnameOverride: "product-sde-postgres"
+  auth:
+    username: "sdeuser"
+    port: 5432
+    database: "sdedb"
+    existingSecret: "sde-postgresql-secrets"
+    secretKeys:
+      adminPasswordKey: "postgres-password"
+      userPasswordKey: "password"
+
+backend:
+  image:
+    repository: ghcr.io/catenax-ng/tx-managed-simple-data-exchanger-backend
+    pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+  imagePullSecrets: []
+
+  nameOverride: ""
+
+  fullnameOverride: ""
+
+  portContainer: 8080
+
+  podAnnotations: {}
+
+  podSecurityContext:
+    fsGroup: 2000
+
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+     drop:
+     - ALL
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 1001
+
+  service:
+    type: ClusterIP
+    port: 7070
+    targetPort: 8080
+
+  backend:
+    endpoints:
+    ## Default api exposing health checks etc
+      default:
+    # -- The network port, which the "default" api is going to be exposed by the container, pod and service
+        port: "7070"
+    # -- The path mapping the "default" api is going to be exposed at
+        path: /backend
+  ingresses:
+    - enabled: false
+      className: "nginx"
+      annotations: {}
+#      kubernetes.io/ingress.class: nginx
+#      kubernetes.io/tls-acme: "true"
+      hostname: ""
+      endpoints:
+        - default
+      tls:
+        enabled: true
+        secretName: ""
+      # -- If present overwrites the default secret name
+      certManager:
+      # -- If preset enables certificate generation via cert-manager namespace scoped issuer
+      # issuer: ""
+      # -- If preset enables certificate generation via cert-manager cluster-wide issuer
+        clusterIssuer: "letsencrypt-prod"
+  #    hosts:
+  #      - chart-example.local
+
+  resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
```

Comment:
CPU, memory requests and limits have to be specified by default.

Reply:
Both frontend and backend

Reply:
resource limits set